### PR TITLE
Suggest a more updated metallb version

### PR DESCRIPTION
**Pull request by fedepaol**
The latest metallb version is configurable via CRs instead of configmap.
Here we suggest using the latest version and we change the configuration
from configmap to CRs.

### CODE REVIEW

**Comment by aojea** [comment]
we have users that are still using old kubernetes versions and I don't know if this can break them, last time we switched the docs here we had several reported issues https://github.com/kubernetes-sigs/kind/issues/2673

Should we wait a few releases of kubernetes more?

**Comment by fedepaol** [comment]
> we have users that are still using old kubernetes versions and I don't know if this can break them, last time we switched the docs here we had several reported issues #2673
> 
> Should we wait a few releases of kubernetes more?

Actually, https://github.com/kubernetes-sigs/kind/issues/2673 was due to the fact that the doc was pointing at the `main` branch where we introduced the breaking change of switching from configmap to crds.

We validate metallb against the supported version of k8s, currently running CI against 1.21 and 1.24 (because 1.25 kind is not out yet :P ). It _should_ work with older versions, but I can't guarantee.

If we don't do this bump though, metallb manifests will fail because they deploy pod security policy which will be rejected in 1.25

**Comment by aojea** [comment]
> If we don't do this bump though, metallb manifests will fail because they deploy pod security policy which will be rejected in 1.25

/ok-to-test

**Comment by fedepaol** [comment]
Reference issue https://github.com/metallb/metallb/issues/1401

File: site/content/docs/user/loadbalancer.md [hunk 1]
```diff
@@ -26,16 +26,10 @@ description: |-
 
 ## Installing metallb using default manifests
 
-### Create the metallb namespace
```
Comment (aojea) [inline site/content/docs/user/loadbalancer.md:29]: Should we add a small note, explaining this history that "since version x metallb moved to CRDs" ...

Comment (fedepaol) [reply site/content/docs/user/loadbalancer.md:29]: will do!

**Comment by BenTheElder** [comment]
thank you!

**Pull request merged**